components/Header.tsx

```diff
@@ -29,57 +48,23 @@ const Header = ({ variant = 'default' }) => {
 
         <div className="items-center flex">
           <ul className="pr-6 sm:flex hidden">
-            <li className="px-6">
-              <Link href="/#jobs">
-                <a
-                  className={classnames({
-                    'text-subprimary': isPrimaryVariant,
-                    'text-gray-800': !isPrimaryVariant,
-                    'font-bold': pathname === '/'
-                  })}
-                >
-                  Jobs
-                </a>
-              </Link>
-            </li>
-            <li className="px-6">
-              <Link href="/#upcoming-events">
-                <a
-                  className={classnames({
-                    'text-subprimary': isPrimaryVariant,
-                    'text-gray-700': !isPrimaryVariant
-                  })}
-                >
-                  Events
-                </a>
-              </Link>
-            </li>
-            <li className="px-6">
-              <Link href="/news">
-                <a
-                  className={classnames({
-                    'text-subprimary': isPrimaryVariant,
-                    'text-gray-700': !isPrimaryVariant,
-                    'font-bold': pathname.startsWith('/news')
-                  })}
-                >
-                  News
-                </a>
-              </Link>
-            </li>
-            <li className="px-6">
-              <Link href="/code-for-vietnam">
-                <a
-                  className={classnames({
-                    'text-subprimary': isPrimaryVariant,
-                    'text-gray-700': !isPrimaryVariant,
-                    'font-bold': pathname.startsWith('/code-for-vietnam')
-                  })}
-                >
-                  Code for Vietnam
-                </a>
-              </Link>
-            </li>
+            {menuItems.map(item => (
+              <li className="px-6">
+                <Link href={item.href}>
+                  <a
+                    className={classnames({
+                      'text-subprimary': isPrimaryVariant,
+                      'text-gray-800': !isPrimaryVariant,
+                      'font-bold':
+                        pathname === item.href ||
+                        (pathname === '/' && item.href === '/#jobs')
```

Comment:
can you check the behavior of the menu on master and try to keep the same behavior here? I don't think the condition check here could fulfill that. Also, I think it is error-prone to hardcode the path like you are doing with `/#jobs` here.